code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -98,13 +105,28 @@
 				update_appearance()
 		else
 			to_chat(user, span_warning("Seems like someone already drew something on this helmet's visor!"))
+		return
+	if(istype(hitting_item, /obj/item/clothing/head))
+		var/obj/item/clothing/hitting_clothing = hitting_item
+		if(hitting_clothing.clothing_flags & PLASMAMAN_HELMET_EXEMPT)
+			to_chat(user, span_notice("You cannot place [hitting_clothing.name] on helmet!"))
+			return
+		if(!attached_hat)
+			attached_hat = hitting_clothing
+			to_chat(user, span_notice("You placed [hitting_clothing.name] on helmet!"))
+			hitting_clothing.forceMove(src)
+			update_appearance()
+		else
+			to_chat(user, span_notice("There's already something placed on helmet!"))
```

Comment:
Move 119-120 to where 114 is and replace with if(attached_hat) and an early return, then you don’t need the current 114 as well as can unindent current 115-118

hate mobile review